bin/release

```diff
@@ -10,5 +11,7 @@ then
     exit 1
 else
     source "${ROOT}/bin/dist-functions"
-    setup "${VERSION}" sdist upload -r pypi --show-response
+    twine upload --repository testpypi \
```

Comment:
The release script here contains the test repository.

Maybe this should be true unless the user passes in a `--real` option or something, whereupon it switches to the real repository.